hathor/cli/run_node.py

```diff
@@ -131,6 +136,46 @@ def prepare(self, args: Namespace) -> None:
 
         python = f'{platform.python_version()}-{platform.python_implementation()}'
 
+        if args.unsafe_mode:
+            unsafe_args_found = []
+            for arg_cmdline, arg_test_fn in self.UNSAFE_ARGUMENTS:
+                if arg_test_fn(args):
+                    unsafe_args_found.append(arg_cmdline)
+
+            message = [
+                'UNSAFE MODE IS ENABLED',
+                '',
+                '********************************************************',
+                '********************************************************',
+                '',
+                'UNSAFE MODE IS ENABLED',
+                '',
+                'You should never use --unsafe-mode in production environments.',
+                '',
+                'The --unsafe-mode parameter allows you to enable unsafe features in your full-node.',
+                '',
+            ]
+
+            fail = False
+            if not unsafe_args_found:
+                message.append('You have not enabled any unsafe feature. Remove the --unsafe-mode and try again.')
```

Comment:
The message may be a bit confusing, what about `'--unsafe-mode is not needed because you have not enabled any unsafe feature. Remove --unsafe-mode and try again.'`

Reply:
Nice suggestion. Done!